### PR TITLE
Fix: #497 Baidu Translator: failed to translate chinses to japanese

### PR DESCRIPTION
**Pull request by zergmk2**
fix the issue Baidu Translator: failed to translate chinses to japanese #497

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/ResXResourceManager?pullRequest=498) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/ResXResourceManager?pullRequest=498) before we can accept your contribution.<br/><br/>:x: zergmk2 [sign now](https://cla.dotnetfoundation.org/dotnet/ResXResourceManager?pullRequest=498)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/ResXResourceManager?pullRequest=498) it.</sub>

File: src/ResXManager.Translators/BaiduTranslator.cs [hunk 1]
```diff
@@ -158,9 +162,17 @@ protected override async Task Translate(ITranslationSession translationSession)
                     }).ConfigureAwait(false);
                 }
             }
+        }
 
-
+        private string GetCultureLang(CultureInfo targetCulture)
+        {
+            if (_baiduCultureMap.ContainsKey(targetCulture.TwoLetterISOLanguageName))
```
Comment (tom-englert) [inline src/ResXManager.Translators/BaiduTranslator.cs:169]: Use TryGetValue

**Review by tom-englert — CHANGES_REQUESTED** [review]

File: src/ResXManager.Translators/BaiduTranslator.cs (hunk 1 above)
Comment (zergmk2) [reply src/ResXManager.Translators/BaiduTranslator.cs]: Done. Thanks.

**Comment by 632575987** [comment]
failed to translate zh-Hans to zh-Hant.

**Pull request merged**